crawl-ref/source/dat/des/variable/mini_monsters.des

```diff
@@ -2317,6 +2317,66 @@ cccc....cccc
 cccccccccccc
 ENDMAP
 
+NAME:   dreamdust_gnoll_farm
+TAGS:   no_monster_gen transparent
+DEPTH:  D:4-7
+COLOUR: pc = brown
+FTILE:  p = floor_mud
+KMONS:  p = fungus / nothing
+MONS:   gnoll ; scythe, hound
+NSUBST: . = 2=1 / 3=1. / 2 / 1=2. / .
+ITEM:   any potion no_pickup
+MARKER: ! = lua:fog_machine { cloud_type='flame', pow_min = 10, pow_max = 10, \
+                              delay = 10, size = 1, excl_rad = 0 }
+MAP
+..............
+.pppppp.ccccc.
+.pWWWWp.c.!dc.
+.pppppp.c...c.
+.pWWWWp.cc+cc.
+.pppppp.......
+..............
+ENDMAP
+
+NAME:   dreamdust_gnoll_shepherd
+TAGS:   no_monster_gen no_item_gen transparent
+DEPTH:  D:7-11, Lair
+KMONS:  1 = gnoll shaman name:gnoll_shepherd \
+            n_rpl n_des n_spe spells:none ; quarterstaff
+KMONS:  2 : yak / dream sheep
+NSUBST: . = 5=2 / .
+MAP
+.....
+..1..
+.....
+ENDMAP
+
+NAME:   dreamdust_looks_like_rain
+TAGS:   transparent no_monster_gen
+DEPTH:  D:13-, Depths
+MONS:   sky beast
+NSUBST: . = 2=1 / 3=1. / .
+KFEAT:  A = stone_arch
+: if you.in_branch("D") then
+KMONS:  A = air elemental / wind drake
+: else
+KMONS:  A = titan / storm dragon
+: end
+: if crawl.coinflip() then
+MARKER: A = lua:fog_machine { cloud_type='rain', pow_min = 10, \
```

Comment:
You can use thunder clouds in Depths, where titans and storm dragons are immune to elec. It might get some collateral kills, but monsters in Depths are stronger, there's only one of this cloud, and we don't care as much if we kill a few stragglers. Seems an appropriate thematic upgrade for Depths, at least.